tools/content-generator/src/test/java/org/projectnessie/tools/contentgenerator/AbstractContentGeneratorTest.java

```diff
@@ -53,7 +53,7 @@ public class AbstractContentGeneratorTest {
   void emptyRepo() throws Exception {
     try (NessieApiV1 api = buildNessieApi()) {
       Branch defaultBranch = api.getDefaultBranch();
-      api.assignBranch().branch(defaultBranch).assignTo(Detached.of(NO_ANCESTOR));
+      api.assignBranch().branch(defaultBranch).assignTo(Detached.of(NO_ANCESTOR)).assign();
```

Comment:
makes me wonder if our api methods that only build but not yet execute an operation should be annotated with:
https://errorprone.info/bugpattern/CheckReturnValue

Reply:
Yes, that'd be nice.

I had an IJ rule for that at some point, but it proved difficult to maintain to match API changes. An annotation would work better, I think.